server/controller/AdminController.js

```diff
@@ -49,6 +49,64 @@ class AdminController {
       });
     }
   }
+
+  /**
+ *@desc approve a requests by an admin
+ *
+ * @param {string} req
+ * @param {string} res
```

Comment:
So req is not a string. For you to use a dot operator on it, what do you think it is?